j/jekyll-commonmark/jekyll-commonmark_ubi_8.5.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : jekyll-commonmark
+# Version       : v1.2.0
+# Source repo   : https://github.com/pathawks/jekyll-commonmark
+# Tested on     : UBI: 8.5
+# Language      : Node
+# Travis-Check  : True
+# Script License: MIT License
+# Maintainer    : Shreya Kajbaje <shreya.kajbaje@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=jekyll-commonmark
+PACKAGE_VERSION=${1:-v1.2.0}
+PACKAGE_URL=https://github.com/pathawks/jekyll-commonmark
+
+yum -y update && yum install -y nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git jq curl make gcc-c++ procps gnupg2 ruby libcurl-devel libffi-devel ruby-devel redhat-rpm-config sqlite sqlite-devel java-1.8.0-openjdk-devel rubygem-rake wget
+
+#yum groupinstall "Development Tools"
+yum install openssl-devel
+wget https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.1.tar.gz
```

Comment:
we can use rvm to install the required ruby version rather than building it from source? (see the templates folder in this repo for reference)